newrelic/detect_test.go

```diff
@@ -73,6 +73,15 @@ func testDetect(t *testing.T, context spec.G, it spec.S) {
 						{Name: "php"},
 					},
 				},
+				{
+					Provides: []libcnb.BuildPlanProvide{
+						{Name: "new-relic-python-config"},
+					},
+					Requires: []libcnb.BuildPlanRequire{
+						{Name: "new-relic-python-config"},
+						{Name: "cpython", Metadata: map[string]interface{}{"build": true}},
```

Comment:
```suggestion
						{Name: "cpython", Metadata: map[string]interface{}{"build": true}},
						{Name: "site-packages", Metadata: map[string]interface{}{"build": true}},
```

Keeps the tests in sync.